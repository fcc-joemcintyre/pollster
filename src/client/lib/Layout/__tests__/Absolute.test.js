/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import { Absolute } from '..';

test ('Absolute renders default component', () => {
  const component = renderer.create (
    <Absolute />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});
