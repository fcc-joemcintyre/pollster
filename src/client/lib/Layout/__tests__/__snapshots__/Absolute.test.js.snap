// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Absolute renders default component 1`] = `
.c0 {
  position: absolute;
}

<div
  className="c0"
/>
`;
